test/hotkey.test.ts

```diff
@@ -151,3 +151,169 @@ test('allows to override hotkeys', () => {
   press(window, { code: 'KeyQ', key: 'й' })
   equal(clicked, 'bqb')
 })
+
+test('should ignore data-keyux-ignore-hotkeys element and call after focus on it', () => {
```

Comment:
```suggestion
test('ignores data-keyux-ignore-hotkeys element and call after focus on it', () => {
```

To keep code style.

Also check that this line is not longer than 80 symbols.